dist/flights/flights.controller.d.ts

```diff
@@ -0,0 +1,27 @@
+import { FlightService } from './flights.service';
+export declare class FlightController {
+    private readonly flightService;
+    constructor(flightService: FlightService);
+    bookFlight(from: string, to: string, date: string, seats: number, flightNumber: string, isBooked: boolean, paymentStatus: string): Promise<{
+        uid: string;
```

Comment:
good to see custom response object , but rather than inline object pls try to create as custom object and can reuse across service and controller response type